main.go

```diff
@@ -233,6 +235,17 @@ func parseCategory(filePath string, categorName string) SharedCategory {
 		toml.Unmarshal(byteValue, &sharedCategory)
 		return sharedCategory
 	}
+	if fileExtension == ".yaml"{
+		yamlFile, err := ioutil.ReadFile(filePath)
+		if err != nil {
+			fmt.Println(err)
+			return SharedCategory{}
+		}
+		byteValue, _ := yaml.Marshal(yamlFile)
+		var sharedCategory SharedCategory
+		yaml.Unmarshal(byteValue, &sharedCategory)
```

Comment:
Unmarshal the `yamlFile` instead of bytes

Reply:
Resolved
and working now